api/functions/getLocation/getLocation.test.js

```diff
@@ -0,0 +1,16 @@
+import { getLocation } from "./getLocation";
+
+describe("getLocation function", () => {
+	describe("when existing crs code given", () => {
+		it("returns matching long&lat value in appropriate format", async () => {
+			const expectedResponse = {
+				location: {
+					latLng: { latitude: 53.47671998, longitude: -2.228977818 },
+				},
+			};
+			expect(await getLocation("MAN")).toEqual(expectedResponse);
```

Comment:
How is this statement coping with floating point rounding issues?

Reply:
We can convert data to string and then check if they match. would this be a solution?

Reply:
No, sadly not. https://floating-point-gui.de/errors/comparison/

You probably want to use expect(...).toBeCloseTo(...)